cli/src/main.rs

```diff
@@ -189,6 +189,7 @@ fn main() {
     }
 }
 
+#[ignore]
```

Comment:
Should this be ignored now?

Reply:
Yes it always fails

Reply:
When I read the workflow files correctly, it was tested before and hasn't failed? I am wondernig what in this PR makes it fail.

Reply:
Ah, there is another #[ignore] here. I'm confused :-)

Reply:
Yes another `#[ignore]` was merged from another branch. I've removed one and kept one ;)